reset-password.py

```diff
@@ -53,10 +53,14 @@ def validate_pin(pin):
 
 if __name__ == "__main__":
     
-    parser = argparse.ArgumentParser(description="Reset an existing user's password")
-    parser.add_argument('username',help='username of the user whose password you wish to reset')
-    parser.add_argument('PIN', type=validate_pin, help='Four-digit PIN provided by the user')
-    parser.add_argument('-c', '--config', 
+    parser = argparse.ArgumentParser(
```

Comment:
Can you change it to use a consident indentation type for all these parser.add_argument lines?  Right now there are 2 different styles:
* lines 56-60 use a newline after the `(` and indent the rest of the lines 4 spaces
* lines 61-64 continue after the opening `(` and line up the rest of the lines with the opening `(` 

These are both fine for PEP8 but we should try to be consistent about which one we use, at least in the same local code block.  I would use the 2nd one if possible because that is how it is in most of the rest of the file.

You can edit the strings to be a little shorter if that would help squeeze things in - sometimes I like when PEP8 forces me to be more concise.